contracts/ccip/interfaces/IGovernanceCCIPReceiver.sol

```diff
@@ -12,22 +12,31 @@ interface IGovernanceCCIPReceiver {
   /// @param messageId The ccip message id.
   /// @param target The target address on the destination chain.
   /// @param payload The payload of the message.
-  event MessageExecuted(
+  event MessageExecutedSuccessfully(
     bytes32 indexed messageId,
     address indexed target,
     bytes payload
   );
 
+  /// @notice Emitted when a message is successfully executed.
+  /// @param messageId The ccip message id.
+  /// @param target The target address on the destination chain.
+  /// @param payload The payload of the message.
+  /// @param _error The error returned by the message call.
+  event MessageExecutionFailed(
+    bytes32 indexed messageId,
+    address indexed target,
+    bytes payload,
+    bytes _error
```

Comment:
@voith I would change the variable name. Usually, you use `_` when the variable is private or won't be used. I would call it just `error` without the `_`

Reply:
`error` is a keyword, I'll rename it to `failure` for consistency.

Reply:
or `errorData`

Reply:
done